database/migrations/2021_11_05_231823_create_permission_tables.php

```
@@ -18,10 +18,10 @@ public function up()
         $teams = config('permission.teams');
 
         if (empty($tableNames)) {
-            throw new \Exception('Error: config/permission.php not loaded. Run [php artisan config:clear] and try again.');
+            throw new Exception('Error: config/permission.php not loaded. Run [php artisan config:clear] and try again.');
```

PHPMD (2.15.0)

Missing class import via use statement (line '21', column '23'). (Clean Code Rules)

(MissingImport)